sycl/include/CL/sycl/info/info_desc.hpp

```diff
@@ -131,7 +131,8 @@ enum class device : cl_device_info {
   usm_shared_allocations = PI_USM_SINGLE_SHARED_SUPPORT,
   usm_restricted_shared_allocations = PI_USM_CROSS_SHARED_SUPPORT,
   usm_system_allocator = PI_USM_SYSTEM_SHARED_SUPPORT,
-
+  // uuid extension
+  ext_intel_device_info_uuid = PI_DEVICE_INFO_UUID,
   // intel extensions
```

Comment:
I think `ext_intel_device_info_uuid` value should go after `// intel extensions` comment. Probably it should be added at the end of intel extensions list.

Reply:
@rbegam, will you resolve this comment?

Reply:
missed it somehow. fixing it.

Reply:
fixed it here.